docusaurus.config.ts

```diff
@@ -33,7 +33,10 @@ const config = {
   favicon: "img/PANW_Parent_Glyph_Red.svg",
   organizationName: "PaloAltoNetworks",
   projectName: "pan.dev",
+<<<<<<< HEAD:docusaurus.config.js
```

Comment:
It looks like there was an issue with a merge, would recommend cleaning up areas like this where it shows changes.